[KOGITO-8031] Kogito Serverless Platform CRD implementation and integration (#41)
kiegroup/kogito-serverless-operator: README.md

```sh 
minikube start --cpus 4 --memory 4096 --addons registry --insecure-registry "10.0.0.0/24"
```
2. Create a namespace for your workflows

```sh
kubectl create namespace kogito-workflows
```

> 3. Create a secret

Reviewer: Please add the purpose of this secret, e.g. pull built images to this registry.etc..

Author: Added an additional description

```sh 
minikube start --cpus 4 --memory 4096 --addons registry --insecure-registry "10.0.0.0/24"
```
2. Create a namespace for your workflows

```sh
kubectl create namespace kogito-workflows
```

3. Create a secret

> This secret it will be used by the Operator to push the images built to the desired docker registry.

Reviewer 2: ```suggestion
This secret will be used by the Operator to push the images built to the desired docker registry.
```

make deploy IMG=<some-registry>/kogito-serverless-operator:tag
```

6. Create a Platform containing the configuration (i.e. registry address, secret) for building your workflows:

You can find a basic Platform CR example in the [config](config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml) folder. 

```sh
kubectl apply -f config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml -n kogito-workflows
```

> 7. Ask to the operator to build and deploy the Greeting workflow:

Reviewer 2: ```suggestion
7. Build and deploy the Greeting workflow with the Operator
```
